tests/test_source.py

```diff
@@ -127,6 +129,15 @@ def test_get_package_hashes_server(self):
              'sha256': '795a1bdddc832289ab958dc9bd9a1c46b849011cbc81fd89d6fe144efc7aae69'}
         ]
 
+        with ZipFile(os.path.join(os.getcwd(), 'tests/data/tensorflow_serving_api-1.13.0-py2.py3-none-any.whl'), 'r') as zip:
+            with tempfile.TemporaryDirectory() as tmpdir:
+                artifact_name = 'tensorflow_serving_api-1.13.0-py2.py3-none-any'
+                zip.extractall(os.path.join(tmpdir, artifact_name))
+                with open(os.path.join(os.getcwd(), 'tests/data/tensorflow_serving_api-1.13.0-py2.py3-none-any.json')) as json_file:
+                    result = json.load(json_file)
+                    source = Source.from_dict(pypi_index)
```

Comment:
This is just initializing source object, we need it because we want to use ```_construct_contents()``` method